src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,82 @@
 
 import org.springframework.stereotype.Service;
 
+/**
+ * Service class providing basic calculator operations.
+ */
```

Comment:
6. **Consistency in Exception Handling**:
   - Ensure consistent exception handling approach throughout the codebase to maintain a unified style and make it easier for developers to understand and maintain the code.